array.c

```diff
@@ -4452,14 +4452,18 @@ ary_recycle_hash(VALUE hash)
  *
  *  Array Difference
  *
- *  Returns a new array that is a copy of the original array, removing any
- *  items that also appear in +other_ary+. The order is preserved from the
- *  original array.
+ *  Returns a new array that is a copy of the original array, removing all
+ *  instances of any item that also appear in +other_ary+. The order is preserved
```

Comment:
Could you avoid the word "instance"?  It is confusing to me with an instance of a class.  How about "all occurrences"?

Reply:
Good idea. Although in this case all the Integers are the same instance but you're right.